imageprocessor-api/src/main/java/pl/edu/uj/kimage/api/StepDependency.java

```diff
@@ -1,16 +1,24 @@
 package pl.edu.uj.kimage.api;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.io.Serializable;
 
 public class StepDependency  implements Serializable {
     private final int dependentStepNumber;
     private final String objectType;
 
-    public StepDependency(int dependentStepNumber, String objectType) {
+    @JsonCreator
+    public StepDependency(@JsonProperty("dependentStepNumber") int dependentStepNumber, @JsonProperty("objectType") String objectType) {
         this.dependentStepNumber = dependentStepNumber;
         this.objectType = objectType;
     }
 
+    public StepDependency(int dependentStepNumber, Class aClass) {
```

Comment:
I think about type, `int` isn't a safe choice. Of course we could assume that it has to be `int` but it should be described